README.md

```diff
@@ -261,6 +261,43 @@ your Jest configuration:
 
 By default Jest ignores everything in `node_modules`. This setting prevents Jest from ignoring the package you're interested in, in this case `@foo`, while continuing to ignore everything else in `node_modules`.
 
+### Test code w/ ES (Webpack 2+) dynamic import
+Newest Webpack introduced ES dynamic import which cannot be handled by babel without plugins even with env preset.
+In development usually use `babel-plugin-syntax-dynamic-import` to prevent errors.
+But in tests it have to be converted to commonjs' require:
+```
+npm install -D babel-plugin-syntax-dynamic-import babel-plugin-dynamic-import-node
+```
+```
+// .babalrc
+{
+  // ...
+  "plugins"      : [
+    "syntax-dynamic-import"
+  ],
+  "env"          : {
+    "test": {
+      "plugins": [
+       "dynamic-import-node"
+      ]
+    }
+  }
+}
+```
+
+Notice, by default `ts-jest` will not use project .babelrc, so `"useBabelrc": true` must be set.
+```json
```

Comment:
I think there's no need to repeat this code snippet again. We can reference the main one here IMO.